### DISCUSSION TITLE
Continuous looping of G-code

### DISCUSSION THREAD

**Discussion by ppkruuv** [mriscoc/Ender3V2S1#996] [Q&A]
Hi

I`m struggling to get looping to work properly on my Ender3V2 with 422 board and it seems that other than M808 not being enabled by default, there hasn´t been too much discussion about it.

What I´ve tried so far:

**M808** - command not recognized at all
**M23 + M24 + M108**  works the first loop but extruder does not start on the second loop, all other axes work as intended
**M32** - command not recognized at all
**Just copy-pasting the same code within a single file** - again, extruder doesn`t start at all during the second loop.

Any suggestions are welcome
Thanks
Peter

**Comment by mriscoc** [comment]
Hi, the g-code M808 was not enabled in the current precompiled versions you must enable GCODE_REPEAT_MARKERS in the configuration_adv.h and build a custom version or request one.
